uplink/metainfo/client.go

```diff
@@ -685,6 +734,12 @@ func (client *Client) DownloadSegment(ctx context.Context, params DownloadSegmen
 		}
 	}
 
+	for i := range response.AddressedLimits {
```

Comment:
why is it clearing the limits?

Reply:
because we don't have pointer on client side anymore when downloading I need to associate `order limit` with `piece num` and because protbuf is not allowing nils on top level I'm sending empty addressed limit which is converted to nil on client side before it will be returned. Earlier we were sorting limits with `sortLimits` method.